nan.h

```diff
@@ -560,6 +560,16 @@ class AsyncResource {
 #endif
 };
 
+inline uv_loop_t* GetCurrentEventLoop() {
+#if NODE_MAJOR_VERSION >= 10 \
+  || NODE_MAJOR_VERSION == 9 && NODE_MINOR_VERSION >= 3 \
+  || NODE_MAJOR_VERSION == 8 && NODE_MINOR_VERSION >= 10
```

Comment:
Can you put the `||` operator on the previous line?  That's more in line with other #if statements in this file.